src/content/en/fundamentals/instant-and-offline/get-started/index.md

```diff
@@ -16,39 +16,49 @@ better apps. Everyone benefits when we write pages that load and run faster, inc
 users. 
 
 Fortunately, many effective techniques exist for improving page speed. While some are bleeding-edge
- innovations that can help shave 9 milliseconds off your load time, there are also many
-  straightforward, high-level paths to achieving significant performance gains, and that's what this
-   document set is about.
+innovations that can help shave 9 milliseconds off your load time, there are also many
+straightforward, high-level paths to achieving significant performance gains, and that's what this
+document set is about.
 
 The processes we used in the past to build and deploy powerful, feature-rich sites now produce web 
 pages that are slow to load and use, especially on mobile devices. Slow-loading pages are
- frustrating for users, who can't/won't use the site, and bad for developers, who forfeit those
-  users. For example:
+frustrating for users, who can't/won't use the site, and bad for developers, who forfeit those
+users. For example, various studies reveal common trends.
 
-* Loading time is a major factor in page abandonment and loyalty; 40% of users report that they 
-abandon sites that take more than two seconds to load, and 79% report that poor-performance sites
- deter them from making purchases (source: 
- [Kissmetrics Blog](https://blog.kissmetrics.com/loading-time/)).
+- Loading time is a major factor in page abandonment and loyalty; 53% of users report that they 
+abandon sites that take more than three seconds to load (source: 
+[SOASTA Google study report](https://soasta.com/blog/google-mobile-web-performance-study/)).
 
-* Users visit more often, stay longer, search more, and buy more frequently on sites that load 
-quickly than on slower ones; a usage reduction of 0.5% (very significant at today's internet traffic
- rates) can occur even when the slowdown is less than half a second (source: 
- [Google Research Blog](https://research.googleblog.com/2009/06/speed-matters.html)).
+- Users visit more often, stay longer, search more, and buy more frequently on sites that load 
+quickly than on slower ones; a usage reduction of 0.5% (very significant at today's internet traffic 
+rates) can occur even when the slowdown is less than half a second (source: 
+[Google Research Blog](https://research.googleblog.com/2009/06/speed-matters.html)).
```

Comment:
Per my previous comment, can we please not reference any statistics that are older than 5 years? The 2009 stat here is still being used. Instead, please find a replacement stat from wpostats.com that works as well or we can alternatively drop this reference/note.